### ISSUE TITLE
[PLUGIN] TinyMCE and <code>

### ISSUE THREAD

**Issue by ange007** [bludit/bludit#813]
### Describe your problem
Invalid formatting ```<code>```.
![chrome_2018-10-04_19-55-05](https://user-images.githubusercontent.com/2736539/46489984-718e0e80-c80f-11e8-91e1-83d8ebf41d2b.png)
(in this screenshot, an attempt to write an article with an example of a plugin)

### Bludit version
3.0

**Comment by ange007** [comment]
Of course, I use this plugin. But nevertheless it does not work as it should.

![image](https://user-images.githubusercontent.com/2736539/46621494-4158af80-cb30-11e8-9004-7d7352449307.png)
![image](https://user-images.githubusercontent.com/2736539/46621586-8381f100-cb30-11e8-87d1-8a752d94f930.png)

But maybe I'm doing something wrong?
Or is something wrong?

![image](https://user-images.githubusercontent.com/2736539/46621627-9f859280-cb30-11e8-9a6c-6673699e1d15.png)

**Comment by clickwork-git** [comment]
Yep, you are right.

**Comment by ange007** [comment]
Is it possible to correct this behavior?

**Comment by dignajar** [comment]
I was testing it and I don't have this problem.

<img width="1047" alt="screenshot 2018-10-22 at 23 17 53" src="https://user-images.githubusercontent.com/507352/47320112-ca9fd400-d650-11e8-962f-d6660db9107f.png">

<img width="1037" alt="screenshot 2018-10-22 at 23 18 55" src="https://user-images.githubusercontent.com/507352/47320162-e99e6600-d650-11e8-9234-db40afba9073.png">

Maybe was some bug in tinymce ? now is updated to v4.8.3.

**Comment by ange007** [comment]
Now i use Bludit 3.2.
The error is in character encoding: ```=>```, ```->``` and maybe other. And you can see it in the source or on the result page.
It is possible that the error and TinyMCE. But most likely something is wrong with the <code> output _(no decoding is used for display)_.

Source:
![image](https://user-images.githubusercontent.com/2736539/47347865-d0c99b00-d6b8-11e8-8f20-e5530f062bdb.png)

Result page:
![image](https://user-images.githubusercontent.com/2736539/47347898-e9d24c00-d6b8-11e8-8f29-2cb848239b8e.png)

**Comment by dignajar** [comment]
I found the problem, is the old code for parsing <pre>.

**Comment by dignajar** [comment]
Update this file, https://github.com/bludit/bludit/commit/5c70cb9fa659cc6ba0161696fadd9623d0053c49
